<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task1 and Task 2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grbRootAquation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Квадратное уравнение</string>
    </property>
    <widget class="QGroupBox" name="grbInData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Входные данные</string>
     </property>
     <widget class="QLineEdit" name="lneRatioA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblRatioA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Коэффициент &quot;A&quot;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneRatioB">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblRatioB">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Коэффициент &quot;В&quot;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneRatioC">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblRatioC">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Свободный член &quot;С&quot;</string>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="cmbCalculateEquation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneResult">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblResult">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbSideTriangle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Расчет стороны треугольника</string>
    </property>
    <widget class="QGroupBox" name="grbInDataTriagle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Входные данные</string>
     </property>
     <widget class="QLineEdit" name="lneSideA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblSideA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Сторона &quot;А&quot;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneSideB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblSideB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Сторона &quot;В&quot;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="grbMeasurement">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>101</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Ед.изм. угла</string>
      </property>
      <widget class="QRadioButton" name="rdbDegrees">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Градусы</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdbRadians">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Радианы</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lneCorner">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblCorner">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Угол</string>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="cmbCalculateTriagle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneResultTriagle">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblResultTriagle">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>132</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
